Makefile

```diff
@@ -31,9 +31,8 @@ build: ## Build the Karpenter KWOK controller images using ko build
 	$(eval IMG_DIGEST=$(shell echo $(CONTROLLER_IMG) | cut -d "@" -f 2))
 	
 apply-with-kind: verify build-with-kind ## Deploy the kwok controller from the current state of your git repository into your ~/.kube/config cluster
-	hack/validation/kwok-requirements.sh
-	kubectl apply -f pkg/apis/crds
-	helm upgrade --install karpenter kwok/charts --namespace $(KARPENTER_NAMESPACE) --skip-crds \
+	kubectl apply -f kwok/charts/crds
+	kubectl apply -f kwok/apis/crds	helm upgrade --install karpenter kwok/charts --namespace $(KARPENTER_NAMESPACE) --skip-crds \
```

Comment:
Is this line correct? Does a kubectl apply with a helm upgrade work?

Reply:
definitely not...... sad.... didn't test this target......